### PR TITLE
Add cup ramen and sinks

### PR DESCRIPTION
**Pull request by Fishfish458**
## About the PR 
This PR:
* Adds ramen as a reagent that water can be added to which makes hot ramen, gives hunger.
* Added to vending machine and bumped up the vending machine counts by 1 (can reverse this if need be, figured 1 person usually needs to eat the entire thing to feed themselves anyhow)
* Added sinks for getting water

**Screenshots**

fill cup

image

image

image

**Changelog**

:cl:
- add: Added insta-heat ramen, just add water!

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Furniture/sink.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  id: Sink
+  parent: SinkEmpty
+  suffix: Water
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      tank:
+        reagents:
+        - ReagentId: Water
+          Quantity: 99999
```
Comment (mirrorcult) [inline Resources/Prototypes/Entities/Structures/Furniture/sink.yml:34]: i get that in ss13 they're infinite but i think i'd prefer if they were at least a little finite

**Review by mirrorcult — APPROVED** [review]
love this, works with the [cooking doc](https://docs.spacestation14.io/en/design/kitchen). in the future when i code solution temperature-based reactions + heating up reagents this can require that as well

File: Resources/Prototypes/Entities/Structures/Furniture/sink.yml (hunk 1 above)
Comment (Fishfish458) [reply Resources/Prototypes/Entities/Structures/Furniture/sink.yml:34]: I figured it's the civie alternative of the chem masters infinite stream except it only outputs water.

What would be a good value?

Comment (Peptide90) [reply Resources/Prototypes/Entities/Structures/Furniture/sink.yml:34]: > I figured it's the civie alternative of the chem masters infinite stream except it only outputs water.
> 
> What would be a good value?

Presumably its a solution container so can be filled from a water tank? I'd say like anything less than 500 max. Presuming the sink is connected to some sort of water tank, idk if 500 is 5L or 50L.

Comment (Fishfish458) [reply Resources/Prototypes/Entities/Structures/Furniture/sink.yml:34]: a bucket holds 500u currently

Comment (Fishfish458) [reply Resources/Prototypes/Entities/Structures/Furniture/sink.yml:34]: Ahh, sure, will put it as 500u for now, figuring out a good solution to the question of 'how much water should it have' will come later.

**Pull request merged**